siliconcompiler/targets/freepdk45_demo.py

```diff
@@ -32,9 +32,9 @@ def setup(chip):
 
     #2. Load PDK, flow, libs combo
     chip.load_pdk('freepdk45')
-    chip.load_flow('lintflow')
+    #chip.load_flow('lintflow')
```

Comment:
why?

Reply:
Thank you! Restored, it was a temporary hack that got checked in by mistake.  Looks like another missing test.